Makefile

```diff
@@ -68,7 +68,6 @@ generate: userconfigs boilerplate imports manifests docs charts fmt ## Run all c
 .PHONY: fmt
 fmt: ## Format code.
 	go fmt ./...
-	trunk fmt
```

Comment:
please change the configuration in `.trunk` instead to exclude `docs` from `markdownlint`:

```yaml
lint:
  enabled:
...
    - markdownlint...
...
  ignore:
    - linters:
        - markdownlint
      paths:
        - docs/**
```